src/components/SearchResults/RequestSearchForm.js

```diff
@@ -25,17 +40,29 @@ export const RequestSearchForm = ({ closeForm, queryTerm = '' }) => {
           pediatric_cancer: '',
           approach: '',
           email: '',
-          email_updates: false
+          email_updates: false,
+          query: queryTerm,
+          request_type: 'search'
         }}
         validationSchema={RequestDataFormSchema}
         validateOnChange={false}
         onSubmit={async (values, { setSubmitting }) => {
-          // TEMP
-          await new Promise((resolve) => {
-            setTimeout(() => resolve(values), 1000)
+          const response = await requestData({
+            requestValues: {
+              ...values
+            }
           })
-          // eslint-disable-next-line no-console
-          console.log(values)
+          // redirects to the homepage after submission
```

Comment:
```suggestion
          // redirect to homepage with notification
```

Reply:
See the commit at ea59aa1f728e6f120e855a60fe45fd11e20cb8cf.